docs/performance/inserts/methods.rst

```diff
@@ -307,3 +311,4 @@ throughput of your cluster with different setups and under different loads.
 .. _translog.durability: https://crate.io/docs/crate/reference/en/latest/sql/reference/create_table.html#translog-durability
 .. _UNNEST reference documentation: https://crate.io/docs/crate/reference/en/latest/sql/statements/insert.html?highlight=unnest#description
 .. _UNNEST: https://crate.io/docs/crate/reference/en/latest/sql/table_functions.html#unnest-array-array
+.. _Overload Protection: https://cratedb.com/docs/crate/reference/en/latest/config/cluster.html#overload-protection
```

Comment:
JFYI: We usually sort those alphabetically, instead of just adding new links at the bottom.